src/main/java/de/uhd/ifi/se/decision/management/jira/model/DecisionKnowledgeProject.java

```diff
@@ -153,47 +153,11 @@ public boolean isPostFeatureBranchCommitsActivated() {
 	}
 
 	/**
-	 * @return uniform resource identifiers of the git repositories for this project
-	 *         as a List<String> (if it is set, otherwise an empty List).
+	 * @return configuration information of the git repositories for this project
+	 *         as a List<GitRepositoryConfiguration> (if it is set, otherwise an empty List).
 	 */
-	public List<String> getGitUris() {
-		return ConfigPersistenceManager.getGitUris(getProjectKey());
-	}
-
-	/**
-	 * @return default branches as Map<String,String> with the uniform resource
-	 *         identifiers of the git repositories for this project as key and the
-	 *         name of default branch as value.
-	 */
-	public Map<String, String> getDefaultBranches() {
-		return ConfigPersistenceManager.getDefaultBranches(getProjectKey());
-	}
-
-	/**
-	 * @return authentification methods as Map<String,String> with the uniform resource
-	 *         identifiers of the git repositories for this project as key and the
-	 *         identifier of the authentification method as value.
-	 */
-	public Map<String, String> getAuthMethods() {
-		return ConfigPersistenceManager.getAuthMethods(getProjectKey());
-	}
-
-	/**
-	 * @return user names as Map<String,String> with the uniform resource
-	 *         identifiers of the git repositories for this project as key and the
-	 *         user names as value.
-	 */
-	public Map<String, String> getUsernames() {
-		return ConfigPersistenceManager.getUsernames(getProjectKey());
-	}
-
-	/**
-	 * @return tokens or passwords as Map<String,String> with the uniform resource
-	 *         identifiers of the git repositories for this project as key and the
-	 *         tokens or passwords as value.
-	 */
-	public Map<String, String> getTokens() {
-		return ConfigPersistenceManager.getTokens(getProjectKey());
+	public List<GitRepositoryConfiguration> getGitConfs() {
```

Comment:
I suggest to use entire names. Please rename into "getGitConfigurations" instead of "getGitConfs"